src/PaginationEmbed.ts

```diff
@@ -120,7 +120,7 @@ export class PaginationEmbed extends MessageEmbed {
    * The embeds if paginating through embeds.
    * @default []
    */
-  public embeds: MessageEmbed[];
+  public embeds: Embed[];
```

Comment:
Instead of passing a generic type we can actually change this type to 
```suggestion
  public embeds: (MessageEmbed|MessageEmbedOptions|APIEmbed)[];
```

Ig it'll solve your issue. But just check which of these types are exported from djs

Reply:
This is true, I should probably patch this around the place - I'll quickly get onto that!